plugins/@grouparoo/bigquery/src/initializers/plugin.ts

```diff
@@ -80,6 +80,14 @@ export class Plugins extends Initializer {
               description: "Private key of service account.",
               placeholder: "e.g. -----BEGIN PRIVATE KEY-----\nMII ...",
             },
+            {
+              key: "location",
+              displayName: "Location",
+              required: false,
+              description:
+                "Region or multi-region code of your dataset.  Required for datasets outside the US.",
```

Comment:
maybe put some expected values in this description (Us, EU, et) to help people.

Reply:
in addition to the placeholder?